### ISSUE TITLE
Is it possible to make clink autocomplete hostnames?

### ISSUE THREAD

**Issue by tomasorti** [mridgers/clink#508]
For example, when using commands like `ping`, `ssh`, `scp`, etc...

Check this question:
https://superuser.com/questions/1395380/autocomplete-host-names-in-windows-cmd

Hosts file is normally found at: `c:\Windows\System32\drivers\etc\hosts`.
So, if `clink` could _know_ which command the user is trying to execute somehow...
(For example, supposedly parsing what is at the start of the line).
Then, if what parses is one of `ping`, `ssh` or similar command, when hitting `<TAB>` would auto-complete what is in the hosts file.

Is it possible to achieve that behaviour in `clink`?

**Comment by DRSDavidSoft** [comment]
Yes, I believe it's possible – though I think reading the `etc\hosts` would require UAC privilege.

You can ask @vladimir-kotikov over at [clink-completions](https://github.com/vladimir-kotikov/clink-completions/issues), maybe he can help you with that.

I'd also 👍 the feature request.
